cmd/oci-create-runtime-bundle/main.go

```diff
@@ -91,6 +91,18 @@ It is strongly recommended to keep the default value.`,
 		&v.version, "version", "v", false,
 		`Print version information and exit`,
 	)
+
+	cmd.SetHelpFunc(func(c *cobra.Command, args []string) {
+		err := cmd.Usage()
```

Comment:
This doesn't show the long description like the standard help function does.  I think we want something like:

```go
origHelp := cmd.HelpFunc()
cmd.SetHelpFunc(func(c *cobra.Command, args []string) {
  origHelp(c, args)
  stdout.Println("\nMore information:")
  stdout.Printf("\tReferences %s\n", image.SpecURL)
  stdout.Printf("\tBug report %s\n", image.IssuesURL)
})
```